tests/tests/qgen.rs

```diff
@@ -217,6 +218,92 @@ async fn generated_single_relation(database: Db) {
     });
 }
 
+///
+/// Property test for GROUP BY aggregates - ensures equivalence between PostgreSQL and bm25 behavior
+///
+#[rstest]
+#[tokio::test]
+async fn generated_group_by_aggregates(database: Db) {
+    let pool = MutexObjectPool::<PgConnection>::new(
+        move || block_on(async { database.connection().await }),
+        |_| {},
+    );
+
+    let table_name = "users";
+    let setup_sql = generated_queries_setup(&mut pool.pull(), &[(table_name, 50)]);
+    eprintln!("{setup_sql}");
+
+    // Columns that can be used for grouping (must have fast: true in index)
+    let grouping_columns = ["name", "color", "age"];
```

Comment:
So, something important to call out is that all of these columns are text, so we're not currently covering grouping by other data types. Can you add a TODO referencing https://github.com/paradedb/paradedb/issues/2903 here?